### DISCUSSION TITLE
External postgres database unreachable from kubernetes pods.

### DISCUSSION THREAD

**Discussion by NagenderSinghRathore** [DefectDojo/django-DefectDojo#8500] [General]
I have been trying to deploy Defectdojo on Kubernetes(AWS EKS) with RDS postgresql as database using helm.

Document referred: https://github.com/DefectDojo/django-DefectDojo/blob/master/readme-docs/KUBERNETES.md

Steps followed:

1. Postgresql secret creation:

`kubectl -n defectdojo create secret generic defectdojo-postgresql-specific --from-literal=postgresql-password=<rds-password> --from-literal=postgresql-postgres-password=<rds-password>`

2. Helm chart installation:

`helm install defectdojo   ./helm/defectdojo --namespace="defectdojo" 
 --set django.ingress.enabled=false --set django.ingress.activateTLS=false 
 --set createSecret=true --set createRabbitMqSecret=true --set createRedisSecret=true --set createMysqlSecret=false  --set createPostgresqlSecret=false --set mysql.enabled=false --set postgresql.enabled=false 
--set postgresql.postgresServer="jdbc:postgresql://<rds-endpoint>:5432"`

Now, I am getting below errors in the celery and django pods:
`django.db.utils.OperationalError: could not translate host name "jdbc:postgresql://<rds-endpoint>:5432" to address: Name or service not known`.

I spun up another pod in the same namespace and able to telnet to the rds-endpoint.

What am I missing here?

Would be really grateful for any help on this.

**Comment by sahil3112** [comment]
Use DD_DATABASE_URL
